Dockerfile

```
@@ -4,12 +4,10 @@ FROM python:3.8-alpine
 # Set the working directory to /app
 WORKDIR /app
 
-# Copy the current directory contents into the container at /app
-COPY requirements /app/requirements
-COPY requirements.txt /app/requirements.txt
+# Install UV
+COPY --from=ghcr.io/astral-sh/uv:latest /uv /uvx /bin/
 
-# Install any needed packages specified in requirements.txt
-RUN pip install --no-cache-dir -r requirements.txt
+COPY uv.lock pyproject.toml /app/
 
```

```suggestion

RUN uv sync --frozen

```

ref: https://docs.astral.sh/uv/guides/integration/docker/#installing-a-project to install deps in the dockerfile.